webapp/src/containers/Rhs/projectDetails/index.tsx

```diff
@@ -71,14 +79,23 @@ const ProjectDetails = (projectDetails: ProjectDetails) => {
 
     const project: FetchSubscriptionList = {project: projectDetails.projectName};
     const {data, isLoading} = usePlugin.getApiState(plugin_constants.pluginApiServiceConfigs.getSubscriptionList.apiServiceName, project);
-    const subscriptionList = data as SubscriptionDetails[];
 
     // Fetch subscription list
     const fetchSubscriptionList = () => usePlugin.makeApiRequest(
```

Comment:
```suggestion
    const fetchSubscriptionList = () => makeApiRequest(
```
And use the destructuring assignment as below - 
```
const {makeApiRequest, getApiState} = usePluginApi();
```